frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -339,7 +339,7 @@ end
 -- to ensure current highlight has not already been cleared, and that we
 -- are not going to clear a new highlight
 function ReaderHighlight:getClearId()
-    self.clear_id = TimeVal.now() -- can act as a unique id
+    self.clear_id = {} -- The pointer address should be fairly unique
```

Comment:
Why play and add uncertainty ? It may _should be fairly unique_ - but it _might not be_ and a bit of memory re-used - may be once a year - and we'll once a year wonder wtf is this bug I can't reproduce :)
I'd rather have a counter, or something that makes it _really unique_.

Reply:
We'd need some *really* unlucky GC behavior to get a collision ^^.

But the reasoning behind the change was flawed anyway (In the heat of the moment, I, err, forgot about the `__eq` and assumed it was essentially doing this instead of a proper comparison ^^), c.f., 0e931d4 ;).